sql-plugin/src/main/301until330-all/scala/com/nvidia/spark/rapids/shims/v2/RapidsErrorUtils.scala

```diff
@@ -30,4 +30,8 @@ object RapidsErrorUtils {
     throw new NoSuchElementException(s"Key: ${elementKey} " +
       s"does not exist in any one of the rows in the map column")
   }
+
+  def preprocessCastFloatToTimestamp(input: ColumnView): Unit = {
```

Comment:
`RapidsErrorUtils` seems like a random class to throw this method into.  I thought the point of this class is to encapsulate how exceptions are thrown once someone has decided to throw an exception rather than house every possible method that might throw on some version of Spark?